projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.spec.ts

```diff
@@ -46,7 +46,9 @@ class TestParentComponent implements OnInit, OnDestroy {
     constructor(public _valueOperationEventService: ValueOperationEventService) { }
 
     ngOnInit() {
-        this.voeSubscription = this._valueOperationEventService.on(Events.ValueAdded, () => this.myNum += 1);
+        this.voeSubscription = this._valueOperationEventService.on(Events.ValueAdded, () => this.myNum = 1);
```

Comment:
Wouldn't it make more sense to add the subscriptions to an array of subscriptions?

Reply:
fixed in dda33f8